exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterTest.java

```diff
@@ -244,4 +250,28 @@ void stringRepresentation() {
               "ZipkinSpanExporter{endpoint=http://zipkin:9411/api/v2/spans, compressionEnabled=false, readTimeoutMillis=15000}");
     }
   }
+
+  @Test
+  void testSuppressInstrumentation() {
+    AtomicBoolean suppressInstrumentation =
```

Comment:
Maybe think of it a different way, make a fake sender, not a real one.

Then pass that to ZipkinSpanExporter ctor.

Then in a suppression closure, do the actual export

After the closure, you can verify sent() and suppressed()

make sense?

```java
class SuppressCatchingSender extends BytesMessageSender.Base {
  AtomicBoolean sent = new AtomicBoolean();
  AtomicBoolean suppressed = new AtomicBoolean();

  SuppressCatchingSender() {
    super(Encoding.JSON);
  }

  @Override public int messageMaxBytes() {
    return 1024;
  }

  @Override public void send(List<byte[]> encodedSpans) {
     sent.set(true);
     suppressed.set(InstrumentationUtil.shouldSuppressInstrumentation(Context.current()));
  }

  @Override public void close() throws IOException {
  }
}
```